PresentScreenings/Views/ColorView.cs

```diff
@@ -6,28 +6,32 @@
 namespace PresentScreenings.TableView
 {
     /// <summary>
-    /// Color view, provides the dedicated color for representation of a screening.
+    /// Color view, provides the dedicated colors for representation of a screening.
     /// </summary>
     /// 
     /// Could be refactored as to work with color-pairs (text-backgound).
 
     public static class ColorView
     {
         #region Private members
-        static NSColor screeningTextColorBlack;
-        static NSColor screeningBgColorBlack;
-        static NSColor screeningTextColorGrey;
-        static NSColor screeningBgColorGrey;
-        static NSColor screeningTextColorDarkGrey;
-        static NSColor screeningBgColorDarkGrey;
-        static NSColor screeningTextColorBlue;
-        static NSColor screeningBgColorBlue;
-        static NSColor screeningTextColorRed;
-        static NSColor screeningBgColorRed;
-        static NSColor screeningTextColorPurple;
-        static NSColor screeningBgColorPurple;
-        static NSColor screeningTextColorAqua;
-        static NSColor screeningBgColorAqua;
+        static NSColor screeningBgColorBlack = NSColor.FromRgb(0, 0, 0);
+        static NSColor screeningTextColorBlack = NSColor.White;
+        static NSColor screeningBgColorGrey = NSColor.FromRgb(219, 219, 219);
+        static NSColor screeningTextColorGrey = NSColor.FromRgb(0, 0, 0);
+        static NSColor screeningBgColorDarkGrey = NSColor.FromRgb(176, 176, 176);
+        static NSColor screeningTextColorDarkGrey = NSColor.FromRgb(0, 0, 0);
+        static NSColor screeningBgColorBlue = NSColor.FromRgb(0, 38, 176);
+        static NSColor screeningTextColorBlue = NSColor.FromRgb(255, 255, 255);
+        static NSColor screeningBgColorRed = NSColor.FromRgb(176, 0, 38);
+        static NSColor screeningTextColorRed = NSColor.White;
+        static NSColor screeningBgColorBrightRed = NSColor.FromRgb(255, 38, 0);
+        static NSColor screeningTextColorBrightRed = NSColor.FromRgb(0, 217, 255);
+        static NSColor screeningBgColorGreen = NSColor.FromRgb(0, 255, 38);
+        static NSColor screeningTextColorGreen = NSColor.FromRgb(255, 0, 217);
+        static NSColor screeningBgColorPurple = NSColor.FromRgb(176, 0, 176);
+        static NSColor screeningTextColorPurple = NSColor.White;
+        static NSColor screeningBgColorAqua = NSColor.FromRgb(38, 255, 176);
+        static NSColor screeningTextColorAqua = NSColor.Black;
```

Comment:
Good! You saved 14 lines of code!
Please consider to make these variables `readonly`.

Reply:
Done.